lib/services/android-plugin-build-service.ts

```diff
@@ -0,0 +1,365 @@
+import * as path from "path";
+import * as shell from "shelljs";
+
+const xml2js = require("xml2js");
+
+export class AndroidPluginBuildService implements IAndroidPluginBuildService {
+
+	constructor(private $fs: IFileSystem,
+		private $childProcess: IChildProcess,
+		private $hostInfo: IHostInfo,
+		private $androidToolsInfo: IAndroidToolsInfo) { }
+
+	private static ANDROID_MANIFEST_XML = "AndroidManifest.xml";
+	private static INCLUDE_GRADLE = "include.gradle";
+	private static MANIFEST_ROOT = {
+		$: {
+			"xmlns:android": "http://schemas.android.com/apk/res/android"
+		}
+	};
+	private static ANDROID_PLUGIN_GRADLE_TEMPLATE = "gradle-plugin";
+
+	private getAndroidSourceDirectories(source: string): Array<string> {
+		const directories = ["res", "java", "assets", "jniLibs"];
+		const resultArr: Array<string> = [];
+		this.$fs.enumerateFilesInDirectorySync(source, (file, stat) => {
```

Comment:
```
return this.$fs.enumerateFilesInDirectorySync(source, (file, stat) => stat.isDirectory() && _.some(directories, (element) => file.endsWith(element)))
```

Reply:
doesn't work, returns true or false i need the array